csharp/src/Google.Protobuf/CodedInputStream.cs

```diff
@@ -845,7 +845,7 @@ public bool MaybeConsumeTag(uint tag)
 
         internal static bool? ReadBoolWrapper(CodedInputStream input)
         {
-            return ReadUInt32Wrapper(input) != 0;
+            return ReadUInt64Wrapper(input) != 0;

```

Comment:
Unrelated to this PR, I think there might be a bug if ReadUInt64Wrapper returns null - I'll try to check that and add a test in a separate PR.

Reply:
Currently it never does so I don't think that'll be a problem.